cachecabinet/Client/src/components/Home/Home.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import { Navigate } from 'react-router-dom';
+
+export default function LoginPage() {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [redirect, setRedirect] = useState(false);
+
+    async function login(ev) {
+        ev.preventDefault();
+        const response = await fetch('http://localhost:4000/login',
```

Comment:
We may need to set that to the env variable so the port is passed in. I am not 100% sure, but in case we know where to look if that does not work on Render